game/log.cc

```diff
@@ -42,6 +45,46 @@
  */
 std::mutex log_lock;
 
+// Capture stderr spam from other libraries and log it properly
+// Note: std::cerr retains its normal functionality but other means of writing stderr get redirected to std::clog
+#if defined(__unix__) || defined(__APPLE__)
+#include <unistd.h>
+#include <future>
+struct StderrGrabber {
+	boost::iostreams::stream<boost::iostreams::file_descriptor_sink> stream;
+	std::streambuf* backup;
+	std::future<void> logger;
+	StderrGrabber(): stream(dup(STDERR_FILENO), boost::iostreams::close_handle), backup(std::cerr.rdbuf()) {
+		std::cerr.rdbuf(stream.rdbuf());  // Make std::cerr write to our stream (which connects to normal stderr)
+		int fd[2];
+		pipe(fd);  // Create pipe fd[1]->fd[0]
```

Comment:
@Tronic getting a warning here that the result is not used. Is this desired behavior?

Reply:
@Baklap4 Fixed in commit 796c5b9c4cb0142e149b0ee6a8a6db6170b57065 of pull request #367.